internal/daemon/controller/controller.go

```diff
@@ -419,10 +423,9 @@ func (c *Controller) Start() error {
 		defer c.tickerWg.Done()
 		c.startCloseExpiredPendingTokens(c.baseContext)
 	}()
-	go func() {
-		defer c.tickerWg.Done()
-		c.started.Store(true)
-	}()
+	if err := c.startWorkerConnectionMaintenanceTicking(c.baseContext, c.tickerWg, c.pkiConnManager); err != nil {
```

Comment:
The pattern for the rest of the tickers is to start it in a goroutine here. Any reason to deviate?